ui/packages/consul-ui/tests/integration/services/routlet-test.js

```diff
@@ -0,0 +1,32 @@
+import { moduleFor, test } from 'ember-qunit';
+moduleFor('service:routlet', 'Integration | Routlet', {
+  // Specify the other units that are required for this test.
+  integration: true,
+});
+test('outletFor works', function(assert) {
+  const routlet = this.subject();
+  routlet.addOutlet('application', {
+    name: 'application'
+  });
+  routlet.addRoute('dc', {});
+  routlet.addOutlet('dc', {
+    name: 'dc'
+  });
+  routlet.addRoute('dc.services', {});
+  routlet.addOutlet('dc.services', {
+    name: 'dc.services'
+  });
+  routlet.addRoute('dc.services.instances', {});
+
+  let actual = routlet.outletFor('dc.services');
+  let expected = 'dc';
+  assert.equal(actual.name, expected);
+
+  actual = routlet.outletFor('dc');
+  expected = 'application';
+  assert.equal(actual.name, expected);
+
+  actual = routlet.outletFor('application');
+  expected = undefined;
+  assert.equal(actual, expected);
+});
```

Comment:
Started out with some basic tests for `outletFor`, this is likely to be added to moving forwards, and potentially the interface may change a little (I saw that instead of passing `(name, route/outlet)` we could just pass `(route/outlet)` and get the name straight off the route/outlet `name` property. I'd rather do that in a refactor PR than a bugfix one though - so that may happen at some point in the future. For now this is just a bit of reassurance testing so I know it does what I need correctly.